packages/e2e/__tests__/servers/server-graphql.ts

```diff
@@ -1,3 +1,4 @@
+import 'reflect-metadata';
```

Comment:
What is `reflect-metadata` being used for in this PR?

Reply:
As I said above, `reflect-metadata` is required for DI, but now it is optional. Then I removed it from the code and introduction :)